integrations/server/test_covidcast.py

```diff
@@ -267,14 +338,63 @@ def test_issues_wildcard(self):
     # make the request
     response = self.request_based_on_row(rows[0], issues="*")
 
-    self.maxDiff = None
     # assert that the right data came back
     self.assertEqual(response, {
       'result': 1,
       'epidata': expected,
       'message': 'success',
     })
 
+  def test_issues_inequality(self):
```

Comment:
suggestion: refactor to remove duplication

The tests for this and `test_time_value_inequality` are super similar. Would it make sense to do something like this instead?

```python
def helper_test_inequality(self, rows, time_field):
    expected = ...
    def fetch(datecode):
        # make the request
        response = self.request_based_on_row(rows[0], **{time_field: datecode})
        return response
    r = fetch(...
    ...

def test_time_value_inequality(self):
    self.helper_test_inequality(self._insert_placeholder_set_six(), "time_value")

def test_issues_inequality(self):
    self.help_test_inequality(self._insert_placeholder_set_seven(), "issue")
```